package.json

```diff
@@ -40,5 +40,10 @@
   "dependencies": {
     "fastify-plugin": "^2.0.3",
     "helmet": "^4.0.0-rc.2"
+  },
+  "tsd": {
+    "compilerOptions": {
+      "esModuleInterop": false
```

Comment:
Aren't we recommeding to have this to `true` to everybody?

Reply:
It is now set to true, and this is only for the way the test file is written. This option can be false and the users have to import the plugin in a different way (I think with import * as whatever from whatever)